templates.go

```diff
@@ -14,7 +14,7 @@ import (
 	"encoding/gob"
 	{{if .ImportFmt}}"fmt"{{end}}
 	"strings"
-	"github.com/mijia/modelq/gmq"
+	"github.com/samingostar/modelq/gmq"
```

Comment:
Should this be `github.com/mijia/modelq/gmq`? Thank you very much.

Reply:
Yeah @mijia 
I've changed it because guessed PR took some time and I couldn' wait for it to use `go get` on my servers. I'll fix it now